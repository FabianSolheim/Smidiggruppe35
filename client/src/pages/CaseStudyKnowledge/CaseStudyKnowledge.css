body {
    margin: 0;
}

.headers {
    position: relative;
    width: 100%;
    height: 600px;
    background-image: url("../../images/knowledgeBackground.png");
    background-repeat: no-repeat;
    background-size: cover;
    background-position: center;
    text-align: center;
    font-size: 64px;
}

.headlines {
    font-size: 64px;
    font-family: "Montserrat", sans-serif;
    color: #FFFFFF;
    padding-top: 150px;
}

.descriptions {
    font-size: 30px;
    font-family: "Montserrat", sans-serif;
    color: #FFFFFF;
    margin-top: 50px;
}

.consultationButton {
    font-family: "Montserrat", sans-serif;
    border-radius: 0;
    height: 50px;
    width: 200px;
    background-color: #034F7A;
    color: white;
    border: 0;
    font-size: 16px;
}

@media only screen and (max-width: 750px) {


    .headlines {
        font-size: 34px;
        margin-top: 100px;
    }

    .descriptions {
        font-size: 16px;
    }

    .consultationButton {
        font-size: 16px;
        margin-top: 60px;
    }

    .headers {
        max-height: 5em;
    }

}
