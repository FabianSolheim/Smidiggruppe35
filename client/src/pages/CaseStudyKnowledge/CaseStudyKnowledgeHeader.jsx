import React from "react";
import "./CaseStudyKnowledge.css";
import { HeaderBox } from "../../components/HeaderBox/HeaderBox";

export function CaseStudyKnowledgeHeader() {
  return (
    <>
      <div className="headers">
        {HeaderBox(
          "../../images/knowledgeBackground.png",
          "Knowledge",
          "white",
          <p>
            More than{" "}
            <strong>
              <em>72 million</em>
            </strong>{" "}
            children of
            <br />
            primary education age are{" "}
            <strong>
              <em>not</em>
            </strong>{" "}
            in school.
            <br />
            <button style={{ margin: 100 }} className="consultationButton">
              <strong>Book a Consultation</strong>
            </button>
          </p>,
          "white"
        )}
      </div>
    </>
  );
}
