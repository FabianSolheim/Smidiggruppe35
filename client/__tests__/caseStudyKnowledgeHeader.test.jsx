import React from "react";
import { CaseStudyKnowledgeHeader } from "../src/pages/CaseStudyKnowledge/CaseStudyKnowledgeHeader";
import ReactDOM from "react-dom";
import { MemoryRouter } from "react-router-dom";

describe("CaseStudyKnowledgeHeader test", () => {
  it("Shows CaseStudyKnowledgeHeader", () => {
    const domElement = document.createElement("p");
    ReactDOM.render(
      <MemoryRouter>
        <CaseStudyKnowledgeHeader />
      </MemoryRouter>,
      domElement
    );
    expect(domElement.innerHTML).toMatchSnapshot();
  });
});
