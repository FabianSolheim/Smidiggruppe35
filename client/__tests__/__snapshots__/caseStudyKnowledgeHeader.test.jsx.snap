// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CaseStudyKnowledgeHeader test Shows CaseStudyKnowledgeHeader 1`] = `"<div class=\\"TopHeader\\" style=\\"background-image: url(https://images.unsplash.com/photo-1522661067900-ab829854a57f?ixlib=rb-1.2.1&amp;ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&amp;auto=format&amp;fit=crop&amp;w=2970&amp;q=80); background-repeat: no-repeat; background-size: cover; background-position: center; text-align: center;\\"><div class=\\"HeaderBox\\"><h1 class=\\"MuiTypography-root MuiTypography-h1 BoxHeadline css-o2w69a-MuiTypography-root\\" style=\\"color: white;\\">Knowledge</h1><div class=\\"BoxText\\" style=\\"color: white;\\"><p>More than <strong><em>72 million</em></strong> children of<br>primary education age are <strong><em>not</em></strong> in school.<br></p></div></div></div><div style=\\"text-align: center; margin-top: -15px;\\"><a class=\\" css-1a6d3yr-MuiButton-root\\" component=\\"button\\" focusvisibleclassname=\\"\\" style=\\"padding: 12px 22px; text-decoration: none; font-weight: 500; border-radius: 0; text-transform: none;\\" classes=\\"[object Object]\\" href=\\"/booking\\">Book a Consultation</a></div>"`;
